### ISSUE TITLE
Create a performance measurement tool through a specific validation set (separate to the actual validation set that is used during training) and TensorFlow tools.

### ISSUE THREAD

**Issue by sd12832** [sd12832/TextBasedMusicClassifier#2]

**Comment by sd12832** [comment]
- [ ] Research and understand measurements inside the NLP field and apply these to this project. 
- [ ] Implement an `eval.py` that takes direction from Tensorboard and other evaluation tools (look into Brightmart's code).
